improve detection of unfollow button

mjishnu opened the pull request:
- improve detection of unfollow button with simpler xpath

mjishnu committed:
improve detection of unfollow button

mr-asady commented:
Thank you for the update! I noticed some issues in the code, which I have resolved. Please review the changes and update if necessary.

mjishnu committed:
Merge branch 'main' into main

mjishnu commented:
> Thank you for the update! I noticed some issues in the code, which I have resolved. Please review the changes and update if necessary.

fixed the merge conflicts btw you forgot to remove your password from the code, i have removed it while clearing the confect but i recommend changing your Instagram password

mjishnu commented on script.py:
```diff
@@ -69,23 +73,26 @@
             try:
-                confirm_div = WebDriverWait(driver, 10).until(
-                    EC.presence_of_element_located((By.XPATH, "//div[@class='x78zum5 xdt5ytf']"))
+                unfollow_button = WebDriverWait(driver, 5).until(
```
this is the change i made

mr-asady committed:
Merge branch 'main' into main

mr-asady commented:
**Thank you so much for letting me know.**
